lua/lspconfig/configs/stylua3p_ls.lua

```diff
@@ -0,0 +1,16 @@
+local util = require('lspconfig.util')
+
+return {
+  default_config = {
+    cmd = { 'stylua-3p-language-server' },
+    filetypes = { 'lua' },
+    root_dir = util.root_pattern('.stylua.toml'),
```

Comment:
can also be a `stylua.toml`: file https://github.com/JohnnyMorganz/StyLua#finding-the-configuration

Reply:
Thanks, fixed